smapply/api.py

```diff
@@ -344,10 +349,13 @@ def parse_webhook_user(webhook):
                     klass_info['id']
                 )
         if personal_price:
-            user.klass_prices.update_or_create(
+            personal_price = user.klass_prices.update_or_create(
                 klass=klass,
                 defaults={'price': personal_price, 'application_stage': application_stage['title']}
-            )
+            )[0]
+            # Sync the personal price
+            from hubspot.task_helpers import sync_hubspot_deal
+            sync_hubspot_deal(personal_price)
```

Comment:
This function `parse_webhook_user` has grown quite a bit. Recommend renaming and breaking it up into separate sub-functions as a code quality improvement.